frontend/src/modules/Catalog/components/Hit.js

```diff
@@ -45,7 +45,7 @@ HitICon.propTypes = {
   hit: PropTypes.object.isRequired
 };
 
-const Hit = ({ hit }) => {
+export const Hit = ({ hit }) => {
```

Comment:
For 2.0.1: We should rename `Hit` to be more descriptive: `SearchHit`? `GlossarySearchResultHit`?

Reply:
I believe `GlossarySearchResultItem` is much more descriptive.